roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RoborazziPreviewScannerSupport.kt

```diff
@@ -14,10 +14,25 @@ fun ComposablePreview<AndroidPreviewInfo>.captureRoboImage(
 ) {
   val composablePreview = this
   composablePreview.applyToRobolectricConfiguration()
-  captureRoboImage(filePath = filePath, roborazziOptions = roborazziOptions) {
-    composablePreview()
-  }
+  captureRoboImage(
+    filePath = filePath,
+    roborazziOptions = roborazziOptions,
+    content = { activityScenario ->
```

Comment:
I think it was practical to add such a captureRoboImage method to allow users to set properties through the activityScenario. Open to other opinions

Reply:
Thanks. Designing these APIs is a little difficult. I'm going to give a talk at a technology conference this Saturday. Sorry for being late, but I would like to consider this over the weekend.